src/SDKs/CognitiveServices/dataPlane/Vision/CustomVision/Training.Tests/BaseTests.cs

```diff
@@ -46,10 +46,6 @@ protected ITrainingApi GetTrainingApiClient()
             ITrainingApi client = new TrainingApi(handlers: HttpMockServer.CreateInstance())
             {
                 ApiKey = TrainingKey,
-#if RECORD_MODE
-                //BaseUri = new Uri("https://deviris2.azure-api.net/V2.0/Training") //new Uri("http://localhost:8080/api_v2.0/training")
-                BaseUri = new Uri("https://cognitivescusppe.azure-api.net/customvision/v2.0/Training")
```

Comment:
Do we still need the `BaseUri` in RECORD_MODE?

Reply:
It depends on how it's recorded, I'm not using it for Prediction so I'm keeping it consistent between Prediction and Training.